server/src/test/scala/controllers/application/ExportPortConfigControllerSpec.scala

```diff
@@ -0,0 +1,102 @@
+package controllers.application
+
+import module.DRTModule
+import org.scalatestplus.play.PlaySpec
+import play.api.test.{FakeRequest, Helpers}
+import uk.gov.homeoffice.drt.crunchsystem.DrtSystemInterface
+import play.api.test.Helpers._
+import uk.gov.homeoffice.drt.testsystem.MockDrtParameters
+
+class ExportPortConfigControllerSpec extends PlaySpec {
+  "ExportPortConfigController" should {
+    "get port config" in {
+      val controller: ExportPortConfigController = exportPortConfigController
+
+      val result = controller.exportConfig.apply(FakeRequest())
+
+      status(result) mustBe OK
+
+      val resultExpected =
```

Comment:
A few formatting issues:
1) There's a blank line at the top, and a bunch of spaces on the last line
2) Some sections have a single blank line between them and some have 2

Let's get rid of the extraneous white space at the top and bottom, and pick either a 1 or 2 line gap between sections